auraescript/aurae.ts

```diff
@@ -0,0 +1,65 @@
+export function print(...values: any[]) {
+    // @ts-ignore
+    Deno.core.print(values.map(toString).join(' ') + "\n");
+}
+
+function toString(value: any): string {
+    if (
+        typeof value === 'object' &&
+        !Array.isArray(value) &&
+        value !== null
+    ) {
+        return JSON.stringify(value, null, 2);
+    } else {
+        return value?.toString();
+    }
+}
+
+type CreateClientDefault = {
+    kind: "default";
+};
+type CreateClientPath = {
+    kind: "path";
+    path: string;
+};
+type CreateClientOpts = {
+    kind: "opts";
+    ca_crt: string;
+    client_crt: string;
+    client_key: string;
+    socket: string;
+};
+type CreateClient =
+    | CreateClientDefault
+    | CreateClientPath
+    | CreateClientOpts;
+
+export function createClient(opts: CreateClient = { kind: "default" }): AuraeClient {
```

Comment:
The return type needs to be wrapped in a `Promise`, so either `Promise<AuraeClient>` or `Promise<number>`, depending on how you handle my feedback in the other comment.